scripts/release.py

```diff
@@ -16,7 +16,7 @@
 import requests
 import setuptools.sandbox
 from twine.commands.upload import upload
-from components import comp_names
+from .components import comp_names
```

Comment:
Are you sure this works?  I would have expected this to fail with a 'relative import attempted outside of a package' but maybe that's a python 2 only error.  You can run the scripts manually without releasing anything to test this by doing, for example:

```
python scripts/release.py --check iotilecore-4.0.0
```

Reply:
Oh, good catch. I assumed it was a package without more than glancing at it (and my IDE complained that it was invalid syntax, which seems like I need to figure out w.r.t. my pycharm config)